_search-plugins/cache-plugins/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Cache types
+parent: Improving search performance
+has_children: true
+nav_order: 100
+---
+
+# Cache types
```

Comment:
Cache types sounds a bit confusing. As this section basically describes different kind of OpenSearch caches which exists as of today which are `on-heap`.
While cache types can also mean on-heap, disk caches to someone reading this, so we call them `cache stores`.

Should we call this section `OpenSearch caches` instead? And add a note explicitly saying that all of them use onHeap stores? And then move on to next section which says `New cache stores`.

What do you think?

Reply:
Can we call this section "Cache stores" or maybe "Caching" or "On-heap caching"? I'd like to avoid the plural "caches" if possible.